app/src/main/java/com/example/tasky/agenda/domain/DateGenerator.kt

```diff
@@ -0,0 +1,9 @@
+package com.example.tasky.agenda.domain
+
+import com.example.tasky.agenda.domain.model.Day
+import java.time.LocalDate
+
+interface DateGenerator {
+    fun getWeek(date: LocalDate = LocalDate.now()): List<Day>
+    fun getMonth(): String
+}
```

Comment:
There's no point in abstracting this, since generating dates doesn't depend on any other APIs from Android or outside. You just need `LocalDate` objects which are totally fine to use in domain (as you can see on the import - it comes from the java package and not from Android)